.github/helm/affine-cloud/templates/secret.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: affine-cloud-secret
+type: Opaque
```

Comment:
And this will be managed by this charts: https://artifacthub.io/packages/helm/bitnami/postgresql